name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java

```diff
@@ -259,6 +260,7 @@ public String getText(Object element)
         portfolioColumns.addColumn(column);
 
         addAttributeColumns(portfolioColumns);
+        addDividendPaymentColumns(portfolioColumns);
```

Comment:
This is on the level of a "portfolio".

I propose *not* to add this. First, the implementation always returns null at the moment. Second, I think the context is missing - which security inside the portfolio has the next dividend date?

Reply:
I agree. Trying to remember what has led me to do this in the first place I can only think of "the end of a long day".